modules/local/h5ad_to_10x.nf

```diff
@@ -0,0 +1,27 @@
+process H5AD_TO_10X {
+    label 'process_low'
+
+    input:
+    path h5ad_file
+    path t2g_file
+
+    output:
+    path "out/*", emit: sample
+
+
+    script:
+    """
+    # convert file types
+    sampleName=`basename ${h5ad_file} .h5ad`
```

Comment:
I think it would be better to get the sample name from `meta`. `input` can take a tuple `meta, h5ad_file` like other processes do.